modular_zubbers/code/modules/storyteller/event_defines/roleset/crew_antagonists/spies.dm

```diff
@@ -4,11 +4,10 @@
 
 	antag_flag = ROLE_SPY
 	antag_datum = /datum/antagonist/spy
-	weight = 4
+	weight = 8
```

Comment:
admittedly spy fucking suck but this is understandable

Reply:
Everything in roleset had its weights raised and split more properly
- 16 weight split 8:8 between spy and traitor
- 10 weight for malf
- 8 weight for BS, Ling, Heretic